package.json

```diff
@@ -407,6 +409,17 @@
               "request": "launch",
               "cwd": ""
             }
+          },
+          {
+            "label": "PowerShell: Debug Runspace",
```

Comment:
Should we mention this applies to the "current" PSIC session.  In other places, we refer to this as the "Interactive Session".  Also, since this is an "attach" it probably should be named something like `PowerShell Attach Interactive Session Runspace`.  Or maybe I'm misunderstanding the "current" process id value?

Reply:
Current was just a way to signal to PSES that we wanna attach to the current process and not some other remote process. I kinda wanted to avoid this all together and just set the process ID to the current PSES process ID but couldn't find a nice clean way to do that. We can call this whatever makes sense. If interactive session makes more sense, let's do it.